testbook/client.py

```diff
@@ -138,12 +138,23 @@ def execute_cell(self, cell, **kwargs) -> Union[Dict, List[Dict]]:
 
         return executed_cells[0] if len(executed_cells) == 1 else executed_cells
 
-    def execute(self) -> None:
+    def execute(self, skip: Optional[Union[slice, list, int]] = None) -> None:
         """
-        Executes all cells
+        Executes all cells besides the ones who were asked to be skipped
         """
-
-        for index, cell in enumerate(self.nb.cells):
+        execute_cells = enumerate(self.nb.cells)
+
+        if skip is not None:
+            if isinstance(skip, int):
+                skip = [skip]
+            
```

Comment:
We would also want to accept cell tag(s) as input for the `skip` argument for users who wouldn't want to concern themselves with knowing which index each cell is at.

```suggestion
            if isinstance(skip, int) or isinstance(skip, str):
                skip = [skip]

            if all(isinstance(x, str) for x in skip):
                skip = [self._cell_index(tag) for tag in skip]

```